packages/nextjs/components/scaffold-eth/Input/EtherInput.tsx

```diff
@@ -122,8 +114,8 @@ export const EtherInput = ({
         >
```

Comment:
Nitpick: 

Above this where there is data-tip maybe use : 

```ts
 data-tip={isNativeCurrenPriceFetching ? "Fetching price" : "Unable to fetch price"}
```



Because if you put this in `page.tsx` : 



```ts
const [etherInputValue, setEtherInputValue] = useState("2");
<EtherInput usdMode value={etherInputValue} onChange={setEtherInputValue} />
```
Lol too much attention to detail, but since we have access to `isNativeCurrenPriceFetching` maybe its worth it ?